app/components/school_information/your_schools/participants_school_search_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SchoolInformation
+  module YourSchools
+    class ParticipantsSchoolSearchComponent < ViewComponent::Base
```

Comment:
Does this relate to participants?

Reply:
Who are participants?

Reply:
ECTs and Mentors

Reply:
It's a school search but with hardcoded participants info for specific cohort